go/vt/vttablet/endtoend/misc_test.go

```diff
@@ -518,3 +520,55 @@ func TestDBAStatements(t *testing.T) {
 		t.Errorf("RowsAffected: %d, want 4", qr.RowsAffected)
 	}
 }
+
+func TestLongQueryErrorTruncation(t *testing.T) {
```

Comment:
In all these tests, it may be simpler and more robust to do equality comparison with the exact output we want.